.env.example

```
@@ -1,44 +1,50 @@
 FRONTEND_URL=http://localhost:3000
 
-# Server Configuration
-PORT=4789
-APP_HOST=localhost
+# ============================================
+# Server Configuration (Optional)
+# ============================================
 
-# Application Configuration
+PORT=4789
+NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)